bin/jenkins.sh

```diff
@@ -5,61 +5,44 @@
 
 # This script makes sure that Jenkins can properly run your tests against your
 # codebase.
-set -e
+set -ex
 
-if [ -z $WORKSPACE ] ; then
-  WORKSPACE=`/bin/pwd`
-fi
-
-cd $WORKSPACE
-VENV=$WORKSPACE/venv
-
-export DB_HOST="localhost"
-export DB_USER="hudson"
+cd /app
+apt-get update
+apt-get install -y libmysqlclient-dev libxml2-dev libxslt1-dev python-dev subversion
 
-echo "Starting build on executor $EXECUTOR_NUMBER..."
 
 # Make sure there's no old pyc files around.
 find . -name '*.pyc' -exec rm {} \;
 
-if [ ! -d "$VENV/bin" ]; then
-  echo "No virtualenv found.  Making one..."
-  virtualenv $VENV --no-site-packages
-  . $VENV/bin/activate
-  pip install --upgrade pip
-  pip install coverage
-fi
-
 git submodule sync -q
 git submodule update --init --recursive
 
-if [ ! -d "$WORKSPACE/vendor" ]; then
-    echo "No /vendor... crap."
-    exit 1
-fi
-
 if [ -d "$WORKSPACE/locale" ]; then
     svn up locale
 else
     svn checkout https://svn.mozilla.org/projects/mozilla.com/trunk/locales/ locale
 fi
 
-. $VENV/bin/activate
-pip install -q -r requirements/compiled.txt
 pip install -q -r requirements/dev.txt
+rm -rf src  # clean up after pip fails to do so
+
+#TODO: get mysql working in new jenkins
+#echo "Creating database if we need it..."
+#echo "CREATE DATABASE IF NOT EXISTS \`${JOB_NAME}\`"|mysql -u $DB_USER -h $DB_HOST
+
+cp bedrock/settings/local.py-dist bedrock/settings/local.py
```

Comment:
Without that line, I was getting an error about a missing local module. I believe that's because the `JENKINS_HOME` env var is not automatically passed in to the container running the tests. I could pass it in explicitly, but I'd like to eliminate settings/jenkins.py entirely and override specific settings with other env variables.